packages/table/README.md

```diff
@@ -0,0 +1,3 @@
+# Table Name
```

Comment:
The documentation is still missing, isn't it?

Reply:
It is I am writing something tomorrow. My muse already left the office. I think this is pretty important since there some questions about the Column and how everything snaps together.